keychain/interface_test.go

```diff
@@ -38,6 +39,13 @@ var (
 		0x4f, 0x2f, 0x6f, 0x25, 0x98, 0xa3, 0xef, 0xb9,
 		0x69, 0x49, 0x18, 0x83, 0x31, 0x98, 0x47, 0x53,
 	}
+
+	// Below are the scrypt parameters that are used when creating the
+	// encryption key for the wallet database with snacl.NewSecretKey.
+	// We use very low values for the tests to speed things up.
+	fastScryptN = 8
```

Comment:
since we use these constants in a number of places, it'd be great if we just exposed them from the `waddrmgr`. definitely not blocking, but would be nice to have eventually

Reply:
Yeah, that would be nice. I'll create a PR there.

Reply:
https://github.com/btcsuite/btcwallet/pull/675

Reply:
that pr has been merged, should be ready to import the newer version and use in this pr :)

Reply:
Added the new version: d501e9a2
This also updates `btcwallet/walletdb` to version `v1.2.0`, I hope that doesn't have any side effects.